tests/unit/utils/env/index.test.ts

```diff
@@ -20,27 +21,78 @@ test('should find a value from a given context', () => {
       context: 'dev' as const,
       value: 'bar',
     },
-  ]
+  ])
   const result = getValueForContext(values, 'dev')
   expect(result).toHaveProperty('value', 'bar')
 })
 
-test('should find a value from a given branch', () => {
-  const values = [
+test('should return a value from the `branch` context with a matching `context_parameter` given a branch', () => {
+  const values = shuffle([
     {
-      context: 'branch-deploy' as const,
+      context: 'branch' as const,
```

Comment:
This test data was invalid, but happened to work because it technically always checks for a matching `context_parameter`